requirements.in

```diff
@@ -24,6 +24,7 @@ python-json-logger
 PyYAML
 redis>=4.4.4
 requests
+respx
```

Comment:
this is a mock library for httpx, a test-only dep. since `mock` is here i figure test-only deps here are fine